src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordClone.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Reflection;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CSharp.Emit;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedRecordClone : SynthesizedInstanceMethodSymbol
+    {
+        public override NamedTypeSymbol ContainingType { get; }
+
+        public SynthesizedRecordClone(NamedTypeSymbol containingType)
+        {
+            ContainingType = containingType;
+        }
```

Comment:
Nit: add a blank line after
```suggestion
        }
        
```